### ISSUE TITLE
Updater app is not working from  8.1.3 to 8.2.2

### ISSUE THREAD

**Issue by ruddyconsult** [owncloud/updater#219]
I have ownCloud Version  8.1.3 (stable) and must update to 8.2.2.

When I log into my admin interface, I see under "Updates" only the following:

A new version is available: {{newVersion}}	

I conclude that the Updater app is not working.

**Comment by VicDeo** [comment]
@ruddyconsult duplicate of https://github.com/owncloud/updater/issues/184
please set `'asset-pipeline.enabled' ` to `false`  in config.php or apply https://github.com/owncloud/updater/pull/198/files

**Comment by ruddyconsult** [comment]
I just now did set 'asset-pipeline.enabled' to false in config.php, and it still doesn't offer me an update.

**Comment by VicDeo** [comment]
@ruddyconsult Does it still show `A new version is available: {{newVersion}}`  in admin ?

**Comment by ruddyconsult** [comment]
Parse error: syntax error, unexpected end of file in
/home/ruddyconsult/websites/owncloud.thomasruddy.org/htdocs/config/config.php
on line 19

**Comment by VicDeo** [comment]
@ruddyconsult it seems you broke something while making changes in config.php. 
May be comma is missed on line 19

**Comment by ruddyconsult** [comment]
Thanks, I had to re-insert the final ;

Now I can get in and see the Update Center!

**Comment by ruddyconsult** [comment]
I tried to do the update, but got this: 
Updating ownCloud to version 8.2.1, this may take a while.
Preparing update
Set log level to debug - current level: "Warning"
Turned on maintenance mode
Exception: Updates between multiple major versions and downgrades are unsupported.
The update was unsuccessful. Please report this issue to the ownCloud community.

**Comment by ruddyconsult** [comment]
I have just noticed that I have two versions: 7.0.4 and 8.1.3 running in parallel. The config.php says 7.0.4. My hoster knows of an 8.1.3 as well.

**Comment by VicDeo** [comment]
@ruddyconsult please check `domain.tld/status.php`. If this is  OC7, not 8.1 you need to migrate to 8.0 and then to 8.1 first 
following the steps from this post https://forum.owncloud.org/viewtopic.php?f=31&t=27723#p84783

**Comment by ruddyconsult** [comment]
{"installed":true,"maintenance":false,"version":"8.2.1.4","versionstring":"8.2.1","edition":""}

**Comment by VicDeo** [comment]
Strange. Is it still shows `Exception: Updates between multiple major versions and downgrades are unsupported.` ?

**Comment by ruddyconsult** [comment]
Yes, I tried again to do an update and got the same msg, starting with "Updating ownCloud to version 8.2.1, this may take a while...."

**Comment by ruddyconsult** [comment]
I have just discovered the parallel instance: parallel to my domain.tld/htdocs there is another folder called domain.tld/2014-05-08 which contains the same OC files! Well, at least files bearing the same names if not content.

**Comment by ruddyconsult** [comment]
It might be a backup, as the config.php file is missing.

**Comment by VicDeo** [comment]
Ok, if the version in config.php is 7.0.4  you need to update manually by repeating the steps below for versions 7.0.12 -> 8.0.10 -> 8.1.5 

1 Download it from https://owncloud.org/changelog/ and extract somewhere. `owncloud` folder will be created
2. Delete all files/folders except:
/data/
/config/
3. Move the files from `owncloud` to your webroot
4. Start the upgrade by opening the webgui or using the occ command line tool (by running `./occ upgrade`)

**Comment by ruddyconsult** [comment]
VicDeo,

OK, it took me all day, but then I got this error message:
Exception: Please upload the ca-bundle.crt file into the 'config' directory.

So I did that, and now 8.1.5 works.

Thanks,



Thomas Ruddy, Germany, http://www.thomasruddy.org/
Privacy in E-Governance http://www.thomasruddy.eu
Seniorexpat@jabber.org
PGP key at
https://pegasus.thomasruddy.org/wp-content/uploads/2014/10/key.htm
Fingerprint: C767 BF04 6226 E790 FFEB
8AFC DD42 D622 0AFF 4D8E

**Comment by ruddyconsult** [comment]
After getting  8.1.5 working (as above), I updated further using the built-in Updater. However I activated an outdated Contacts app, and now my Owncloud site will not open at all. So I restored using the most recent Updater backup.  Owncloud will still not open. Please advise.

**Comment by VicDeo** [comment]
@ruddyconsult what is the current state?
Start with moving `contacts` directory out of `apps` directory.

**Comment by ruddyconsult** [comment]
@VicDeo 
Many thanks! The web version opens now.
